diff --git a/node_modules/ramda-adjunct/types/index.d.ts b/node_modules/ramda-adjunct/types/index.d.ts
index 6060e4a..9b3ade7 100644
--- a/node_modules/ramda-adjunct/types/index.d.ts
+++ b/node_modules/ramda-adjunct/types/index.d.ts
@@ -545,8 +545,8 @@ declare namespace RamdaAdjunct {
          * keys renamed according to the keysMap object as `{oldKey: newKey}`.
          * When some key is not found in the keysMap, then it's passed as-is.
          */
-        renameKeys(keysMap: Dictionary<string>, obj: object): object;
-        renameKeys(keysMap: Dictionary<string>): (obj: object) => object;
+        renameKeys(keysMap: Dictionary<string>, obj: Record<string, unknown>): Record<string, unknown>;
+        renameKeys(keysMap: Dictionary<string>): (obj: Record<string, unknown>) => Record<string, unknown>;
 
         /**
          * Creates a new object with the own properties of the provided object, but the
